extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -2588,6 +2588,87 @@ async function testAudit(path: string, uri: string, condition: typeof expect | t
   expect(auditRecords.records).toEqual(condition.arrayContaining([expect.objectContaining({ type: 'error' })]));
 }
 
+test('activate on mac register commands for setting compatibility moide ', async () => {
+  vi.mocked(isMac).mockReturnValue(true);
+  vi.mocked(isWindows).mockReturnValue(false);
+  vi.mocked(isLinux).mockReturnValue(false);
+  vi.spyOn(PodmanInstall.prototype, 'checkForUpdate').mockResolvedValue({
+    hasUpdate: false,
+  } as unknown as UpdateCheck);
```

Comment:
This could have side effect on the other tests ?

Reply:
not more than the other https://github.com/containers/podman-desktop/blob/a230b9d8c2c3102df3df01955f4ac5134a860ca9/extensions/podman/packages/extension/src/extension.spec.ts#L2532

but anyway there is
https://github.com/containers/podman-desktop/blob/a230b9d8c2c3102df3df01955f4ac5134a860ca9/extensions/podman/packages/extension/src/extension.spec.ts#L176

so having all mocks being reset it should not impact any other test